css/cssom-view/scrollIntoView-vertical-rl-writing-mode.html

```diff
@@ -68,8 +68,20 @@
 // As browsers differ in the meaning of scrollLeft when
 // in a right-to-left mode, we adjust the expectation
 // to match the semantics of scrollLeft.
-if(scroller.scrollLeft === 0)
-  expectedX = [ -box_width, -(((3*box_width - scroller_width)/2)+ (scrollbar_width/2)), -(((2*box_width)-scroller_width)+scrollbar_width)];
+// As browsers differ in the meaning of scrollLeft when
+// in a right-to-left mode, we adjust the expectation
```

Comment:
I think the x-coordinate is nonpositive when box has leftward overflow direction:
https://drafts.csswg.org/cssom-view/#scroll-an-element
and increases rightwards https://drafts.csswg.org/cssom-view/#scrolling-area-origin
so the initial value is 0 per the spec.

So what is calcutated here should actually be the initial value of expectedX and you should test for scroller.scrollLeft  > 0 here to fix the behavior for browsers that don't follow the spec.

nit: please add a space after "if"